### PR TITLE
Release 14.3.1

### PR DESCRIPTION
**Pull request by camscale**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,7 +9,7 @@ require (
 	github.com/alecthomas/kong v0.2.22
 	github.com/go-resty/resty/v2 v2.11.0 // indirect
 	github.com/gogo/protobuf v1.3.2
-	github.com/google/uuid v1.3.1
```
Comment (fheinecke) [inline go.mod:12]: Are the changes to this file intentional? This doesn't usually get changed on release

**Review by fheinecke — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (camscale) [reply go.mod:12]: Huh? It's been changed on at least the last 5 releases. Go code gets updated as I understand it as the terraform provider has parts generated which also depends on the teleport API, so we probably get mod changes from there too.

I just ran the usual `make update-version VERSION=...` and raised the PR on its changes.

Comment (fheinecke) [reply go.mod:12]: Ah sorry, I've just missed that on previous PRs. There are enough go.mod/go.sum changes in this one that it really stood out.

Comment (camscale) [reply go.mod:12]: Most of it seems to be in the second `requires` section, which are the indirect dependencies. Something we depend on depends on a whole lot less!

**Pull request merged**